File: Week 6/mario.py
num = input("Height: ")

while(True):
    if(num.isdigit() == True and int(num) >= 1 and int(num) <= 8):
        break
    else:
        num = input("Height: ")

num = int(num)
for i in range(1,num+1):
    for j in range(0,num-i):
        print(" ",end = "")
    for j in range(0,i):
        print("#",end = "")
    print()

File: Week 1/mario.c
#include<stdio.h>

int main(){
    int num;
    printf("Height : ");
    scanf("%d",&num);
    while(num <= 0){
        printf("Height : ");
        scanf("%d",&num);
    }
    for (int i = 1; i <= num; i++){
        for(int j = num-i; j > 0 ;j--){
            printf(" ");
        }
        for(int j = 0; j < i; j++){
            printf("#");
        }
        printf("\n");
    }
}